ako-operator/bundle/manifests/ako-operator.clusterserviceversion.yaml

```diff
@@ -99,8 +100,8 @@ metadata:
     operators.operatorframework.io/builder: operator-sdk-v1.16.0+git
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     operatorframework.io/suggested-namespace: avi-system
-    containerImage: projects.registry.vmware.com/ako/ako-operator@sha256:cdebc910497c3eaf3624c84fd9e27600b13b53cf3142d098dff0742cb16c624b
-  name: ako-operator.v1.8.2
+    containerImage: projects.registry.vmware.com/ako/ako-operator@sha256:ad55b86975cd73dfd5b3ff1179c03a6cfe47f578c5c8d79ad37d5ce2aab6482b
+  name: ako-operator.v1.9.1
```

Comment:
version should be 1.9.3?

Reply:
Have not made the bundle changes yet. Will add it in the next PR with updated `sha` for ako-operator:1.9.3 image once this PR is merged and image is generated.